spec/features/create_mediated_page_request_spec.rb

```diff
@@ -164,7 +164,7 @@ def fill_in_required_fields
       min_date = date_input['min']
       date_input.set(min_date)
     else
-      wait_for_ajax
+      sleep 1 # The fetch request is about to update #request_needed_date, so give it time to do so.
```

Comment:
i went looking for info about `wait_for_ajax` and found this [somewhat interesting discussion on `wait_for_pending_requests` in capybara](https://github.com/teamcapybara/capybara/issues/2106). tl;dr there isn't currently (and won't be) a method like `wait_for_ajax`, so our options are:
1. do the above (just `sleep`)
2. add an actual class or some other change to the UI so that we know when the change has completed, and then use `.has_css?` which will wait until the change appears

the second option seems like the "nicer" one, but I guess I'm ok with this as a stopgap.

Reply:
@thatbudakguy yes it's really a design question.  How does the user know that something changed?   They obviously aren't going to be looking in the developer console and saying "Hey, it looks like those XHRs finished."  That's the equivalent of this `wait_for_ajax` method.